components/colors.js

```diff
@@ -0,0 +1,12 @@
+export default {
+    white: 'rgba(255, 255, 255, 1)',
+    black: 'rgba(0, 0, 0, 1)',
+    lightGray: 'rgba(234, 234, 234, 1)',
+    darkGray: 'rgba(76, 80, 85, 1)',
+    gray: 'rgba(139, 144, 147, 1)',
+    gradientColor1: 'rgba(223, 122, 153, 1)',
```

Comment:
Maybe a more describing name would be suitable here? I think this is pinkish, while the other one is kinda yellow.

Reply:
Yeah, just named them that because it's the only place it's used.

Reply:
Should be updated now 😎